.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## What is this change?
+- (List out the changes.)
+- (Link to any relevant Github issues or Slack discussion)
+
+## Considerations for discussion
+- (List out any significant design decisions that were made and why.)
+
+## How to test the changes
+- (How should a reviewer test this functionality)
```

Comment:
Sometimes tests are not needed (ie for most docs changes) so maybe add an 'if possible/needed' to that part of the template?

Other than that, this looks good!

Reply:
looks great - thanks so much for doing this!